code/game/atoms_movable.dm

```diff
@@ -363,3 +363,70 @@
 		target.fingerprints += fingerprints
 		target.fingerprintshidden += fingerprintshidden
 	target.fingerprintslast = fingerprintslast
+
+/atom/movable/proc/do_attack_animation(atom/A, visual_effect_icon, obj/item/used_item, no_effect, end_pixel_y)

```

Comment:
for some reason, TG got rid of `end_pixel_y`. I re-included it; without it, attacks were super quick and copy, rather than a smooth move towards target, move back to original position.

It was more "twitch to target, twitch back to original position".

It was trivial to re-include it; dunno why it was gotten rid of to begin with.